mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClient.java

```diff
@@ -59,6 +59,13 @@ public Run getRun(String runUuid) {
     return mapper.toGetRunResponse(httpCaller.get(builder.toString())).getRun();
   }
 
+  public List<Metric> getMetricHistory(String runUuid, String key) {
```

Comment:
Named the run ID parameter `runUuid` for parity with the existing Java `getRun` etc APIs, but we should consider renaming the `runUuid` argument to `runId` in Java APIs to match Python (cc @aarondav)

Reply:
Got it/